src/cats/cats.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller, Post, Body } from '@nestjs/common'
+import { CreateCatDto } from './dto/create-cat.dto'
+import { CatsService } from './cats.service'
+
+@Controller('cats')
+export class CatsController {
+  constructor(private catsService: CatsService) {}
+
+  @Post()
```

Comment:
@shogo-mitomo 
I found this example repo by `NestJS` which adds more decorators for `response`, etc.
https://github.com/nestjs/nest/blob/master/sample/11-swagger/src/cats/cats.controller.ts#L20

Could you add them as well to our repo. :bowing_man: